extension.yaml

```diff
@@ -0,0 +1,173 @@
+# Copyright 2021 Meilisearch
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Identifier for your extension
+name: firestore-meilisearch-search
+
+# Follow semver versioning
+version: 0.1.1
```

Comment:
Don't you want the v`0.1.0` instead ?

Reply:
I really can't name this first version 😊